Assets/Scripts/Gameplay/Action/ConcreteActions/LaunchProjectileAction.cs

```diff
@@ -1,34 +1,35 @@
+using System;
 using Unity.Netcode;
 using UnityEngine;
 using BossRoom.Scripts.Shared.Net.NetworkObjectPool;
 using Unity.Multiplayer.Samples.BossRoom.Server;
+using Unity.Multiplayer.Samples.BossRoom.Visual;
 
 namespace Unity.Multiplayer.Samples.BossRoom.Actions
 {
     /// <summary>
     /// Action responsible for creating a projectile object.
     /// </summary>
+    [CreateAssetMenu()]
     public class LaunchProjectileAction : Action
```

Comment:
mmm the name might be misleading vs the pure FX projectile. "Launch GameObject projectile"? could be grat to have an opposition vs the other FX projectile action we have.

Reply:
I think that this name is ok - it actually works with go's that have `NetworkProjectileState`